www/index.html

```diff
@@ -56,32 +81,90 @@
 			</div>
 		</div>
 	</section>
+	<script src="/src-min/ace.js"></script>
```

Comment:
I believe the better approach is to include a CDN hosted file that includes the whole `ace.js` bundle, and to avoid adding all the files (400 or so) to this repository.
For example, you could use
https://cdnjs.cloudflare.com/ajax/libs/ace/1.4.6/ace.js

Reply:
Okay I'll do this. thanks.

Reply:
I already set it up to work with the CDN

Reply:
@harlev thats a good suggestion.